html/objects/_flag.scss

```diff
@@ -7,45 +7,28 @@
 ////
 
 .sprk-o-Flag {
-  display: table;
-  width: 100%;
-}
-
-.sprk-o-Flag__body,
-.sprk-o-Flag__figure {
-  display: table-cell;
-  vertical-align: top;
+  display: flex;
+  align-items: flex-start;
 }
 
 .sprk-o-Flag__figure {
   padding-right: $sprk-flag-gutter;
-
-  > img {
-    display: block;
-    max-width: none;
-  }
 }
 
 .sprk-o-Flag__body {
-  width: 100%;
+  flex: 1;
```

Comment:
flex: 1, the shorthand, has issues in ie11 sometimes. Can you take a look there.